canonical-kubernetes/addons/jfrog/README.md

```diff
@@ -0,0 +1,152 @@
+# JFrog Artifactory Enterprise Addon to Canonical-Kubernetes Spell
+
+Select this Addon to install Artifactory Enterprise on a Canonical Distribution of Kubernetes (CDK) Cluster in the cloud (currently AWS only) and region you have already selected. After completing this installation JFrog Artifactory will be installed on your Kubernetes cluster. Once you apply the Artifactory licenses you will be able to use Artifactory as a Docker registry, Helm repo, NPM repo and more.
+
+## Prerequistes:
+
+- AWS:- Amazon AWS Account setup with all priveleges required to create a Kubernetes Cluster, create persistent storage, and run load balancers.
+
+- AWS Secret and Key available.
+
+- Installation machine (where you install and run conjure-up)
+  -  sudo privielges
+  -  OS is certified by JFrog(http://www.jfrog.com) and Canonical(http://wwww.canonical.com)(e.g Ubuntu 16.04 LTS).
+  -  Conjure-up dependenices installed (python, jq, perl)
+
+
+- conjure-up installed. Please see http://conjure-up.io for more details on conjure-up. Please note that JFrog has tested using the Locally Hosted Controller option. Juju as A Service (Jaas) option has not been cerfified by JFrog.
```

Comment:
May want to say something like

`JFrog is tested against the self hosted controller`

Reply:
+1